tests/jquery.flot.navigate.Test.js

```diff
@@ -31,18 +31,38 @@ describe("flot navigate plugin", function () {
         expect(typeof plot.zoomOut).toBe('function');
         expect(typeof plot.pan).toBe('function');
         expect(typeof plot.smartPan).toBe('function');
-
-        plot.shutdown();
     });
 
     it('shuts down the plots after use', function() {
         plot = $.plot(placeholder, [
-            []
+            [
+                [0, 0],
+                [10, 10]
+            ]
+        ], options);
```

Comment:
For this test the data is not relevant and it's taking too many lines. For the readability sake create the plot on a single line.